.github/workflows/android.yml

```diff
@@ -143,3 +143,21 @@ jobs:
         with:
           fail_ci_if_error: true
           verbose: false
+  report:
+    needs: build
+    runs-on: ubuntu-latest
```

Comment:
Do we really need to run this report on a separate build flow (build machine to be exact) and can't just run it in the same flow?

Reply:
I think there is something connected to permission. That why it needed to split to 2 workflow, as suggested in the readme of the dev github test reporter.

Reply:
That why it cannot be on the same workflow, even as I did here:
`Resource not accessible by integration`